extension/js/agent/patches/patchMarionette.js

```diff
@@ -37,19 +37,22 @@ this.patchMarionette = (function(agent) {
     }
 
     this.patchMarionetteApplication(Marionette.Application);
-    this.patchMarionetteBehavior(Marionette.Behavior);
+    if(Marionette.Behavior) {
+      this.patchMarionetteBehavior(Marionette.Behavior);  
+    } 
     this.patchMarionetteModule(Marionette.Module);
     this.patchMarionetteController(Marionette.Controller);
 
-    _.each(
-      [
-        Marionette.Application.prototype, Marionette.Module.prototype,
-        Marionette.Behavior.prototype, Marionette.Region.prototype,
-        Marionette.Controller.prototype
-      ],
+    var marionetteClasses = [
```

Comment:
So, this is weird:

+  why patch some prototypes and not others
+ why patch some things twice

you'd think I'd know/remember, but I dont. Honestly, I think it was all accidental...


Mind checking if we can patch everything once and never do the prototype?